### PR TITLE
Rename relaxer and suggester

### PR DESCRIPTION
**Pull request by cuixq**
This PR addresses https://github.com/google/osv-scanner/pull/817#issuecomment-1977729821.

Package `relaxer` is renamed to `relax` and package `suggester` is renamed to `suggest`.
The getter and options are also renamed to avoid duplicates in their names.

### CODE REVIEW

File: internal/remediation/relax.go
Comment (michaelkedar) [inline internal/remediation/relax.go:1]: Thoughts on renaming this file to `relock_relax.go` to make it more distinct from `relax/...`?

**Review by michaelkedar — APPROVED** [review]
LGTM

**Review by G-Rath — APPROVED** [review]

File: internal/remediation/relax.go
Comment (cuixq) [reply internal/remediation/relax.go:1]: SGTM

File: internal/remediation/relax/relax.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ type RequirementRelaxer interface {
 	Relax(ctx context.Context, cl resolve.Client, req resolve.RequirementVersion, allowMajor bool) (resolve.RequirementVersion, bool)
 }
 
-func GetRelaxer(ecosystem resolve.System) (RequirementRelaxer, error) {
+func Get(ecosystem resolve.System) (RequirementRelaxer, error) {
```
Comment (another-rex) [inline internal/remediation/relax/relax.go:26]: Now that it's called relax / suggest, we probably can keep it as GetRelaxer:
```
relaxer := relax.GetRelaxer(eco)
// vs
relaxer := relax.Get(eco)
```

Comment (another-rex) [reply internal/remediation/relax/relax.go:26]: But I don't feel strongly about it either way, so up to you, happy to merge it as is.

**Review by another-rex — APPROVED** [review]

File: internal/remediation/relax/relax.go (hunk 1 above)
Comment (oliverchang) [reply internal/remediation/relax/relax.go:26]: +1. I think `.Get()` would make sense when the package name is what is being gotten.

Comment (cuixq) [reply internal/remediation/relax/relax.go]: done!

**Review by another-rex — APPROVED** [review]

**Pull request merged**